src/components/descricaoProfs/descricaoProfs.style.jsx

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+
+export const TextNomeProf = styled.a`
+    font-family: Quicksand;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 36px;
+    line-height: 40px;
+    margin: 0px;
+    color: #000000;
+`;
+
+export const TextTitle = styled.a`
+    font-family: Quicksand;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 28px;
+    line-height: 30px;
+
+    color: #000000;
+`;
```

Comment:
O elemento `<a>` em html é utilizado para redirecionar para outras páginas. Não sei se entendi mto bem o uso aqui, vc quer usar o elemento para redirecionar para outra página ou é um texto "puro"?
Se for puro usa o elemento html styled.`<span>`